modules/mauth-proxy/README.md

```diff
@@ -6,29 +6,29 @@ The service appends valid mAuth headers to incoming requests and
 sends the authenticated request to the requested service.
 
 ## Running via sbt:
+For Bash:
 ```bash
 export APP_MAUTH_UUID="<the mauth id of the app you are proxying to>"
 export APP_MAUTH_PRIVATE_KEY="<the actual key (not the path) of the app you are proxying to>"
 ./runMauthProxyServer.bash
 ```
 
+For Powershell:
+```posh
+$env:APP_MAUTH_UUID="<the mauth id of the app you are proxying to>"
+$env:APP_MAUTH_PRIVATE_KEY="<the actual key (not the path) of the app you are proxying to>"
+./runMauthProxyServer.ps1
+```
+
 ## Running via Docker image
 
 Internal Medidata users please follow [internal guide](https://mdsol.jiveon.com/community/research-development/app-dev/platform-development/services/mauth/blog/2018/03/06/use-mauthed-service-with-a-mauth-unaware-client)
 
 ### Installing Docker
 You need to have Docker installed on your machine in order to use Mauth Proxy. The simplest way to get Docker running 
 on a Mac or Windows machine is to follow the instructions at 
-[Docker for Mac](https://docs.docker.com/#/docker-for-mac).
-
-You can create a default VM for docker and run it by running the following
-commands:
-
-```bash
-docker-machine create --driver virtualbox default
-docker-machine start
-eval $(docker-machine env)
-```
+[Docker for Mac](https://docs.docker.com/#/docker-for-mac) for Mac and 
+[Docker for Windows]https://store.docker.com/editions/community/docker-ce-desktop-windows for Windows.
```

Comment:
This markdown link is missing parentheses.

I'm surprised the links for Mac and Windows are so different, but :man_shrugging: